src/wai_data_tools/config.py

```diff
@@ -5,15 +5,15 @@
 import yaml
 
 
-def load_config(config_filepath: pathlib.Path) -> Dict[str, Any]:
+def load_config(config_file: pathlib.Path) -> Dict[str, Any]:
     """Load a config YAML file and returns content.
 
     Args:
-        config_filepath: Path to config file
+        config_file: Path to config file
 
     Returns:
         Content of config file
     """
-    with config_filepath.open(mode="r") as f:
-        content = yaml.safe_load(f)
+    with config_file.open(mode="r") as file:
```

Comment:
I meant the other way around, should have added an example:
```
with config_filepath.open(mode="r") as config_file:
```

Becaues we could open other files, and the open makes sense to open the filepath.

Reply:
ah my mistake haha, I'll roll back the changes and rename file to config_file :)